bin/p2-wipe-reality/main.go

```diff
@@ -19,6 +21,16 @@ a known state.
 `
 )
 
+func handleError(err error, message string) {
```

Comment:
I can see the benefit in wanting to unpack the hidden error messages. But:

1. What if I eventually want to do something other than print the error? Can you unlock without side effects?
2. This is unrelated to adding TLS flags. Can you put it in its own commit?